addons/t9n/static/src/core/language_list.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<templates>
+    <t t-name="t9n.LanguageList">
+        <div class="container mt-3">
+            <div class="input-group">
+                <div class="form-outline">
+                    <input type="text" class="form-control" placeholder="Search languages" t-model="state.filters.searchText"/>
+                </div>
+            </div>
+            <table class="table table-hover">
+                <thead>
+                    <tr>
+                        <th t-on-click="() => this.onClickColumnName('name')">Language Name <i t-attf-class="btn btn-link fa fa-sort-{{this.state.sorting.column === 'name' and this.state.sorting.order === 'asc' ? 'up' : 'down'}}"></i>
+                        </th>
+                        <th>Native name</th>
+                        <th>Locale</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <t t-foreach="this.languages" t-as="language" t-key="language.id">
+                        <tr>
+                            <td>
+                                <button class="btn btn-link " t-on-click="() => this.onClickLanguage(language.id)">
+                                    <t t-esc="language.name"/>
+                                </button>
+                            </td>
+                            <td t-esc="language.native_name"/>
+                            <td t-esc="language.code"/>
+                        </tr>
+                    </t>
```

Comment:
(maybe out of the scope of this PR)
If there is no language, it would be nice to have an explanatory text and a button to add more languages